webapp/src/components/MessageList.tsx

```diff
@@ -0,0 +1,62 @@
+'use client';
+
+import ListItem from '@mui/material/ListItem';
+import { FC, useCallback } from 'react';
+import { FixedSizeList, ListChildComponentProps } from 'react-window';
+
+import MessageForm from '@/components/MessageForm';
+import { MessageData } from '@/utils/adapters';
+
+type MessageListProps = {
+  languageName: string;
+  messages: MessageData[];
+  projectName: string;
+  saveTranslation: (
+    projectName: string,
+    languageName: string,
+    messageId: string,
+    translation: string,
+  ) => Promise<void>;
+  translations: Record<string, string>;
+};
+
+const MessageList: FC<MessageListProps> = ({
+  languageName,
+  messages,
+  projectName,
+  saveTranslation,
+  translations,
+}) => {
+  const renderRow = useCallback(
+    (props: ListChildComponentProps): JSX.Element => {
+      const { index, style } = props;
+      const message = messages[index];
+      return (
+        <ListItem key={index} component="div" disablePadding style={style}>
+          <MessageForm
+            languageName={languageName}
+            message={message}
+            projectName={projectName}
+            saveTranslation={saveTranslation}
+            translation={translations[message.id]}
+          />
+        </ListItem>
+      );
+    },
+    [languageName, messages, projectName, saveTranslation, translations],
+  );
+
+  return (
+    <FixedSizeList
+      height={window.innerHeight}
+      itemCount={messages.length}
+      itemSize={200}
```

Comment:
Have set this to a fixed size in https://github.com/zetkin/lyra/pull/107/commits/3db3fd70deed776d84ac7cafd4c59b140fa4d1b8. Until we make time to look at it in detail, I think it's better to have a UX limitation like that rather than the false flexibility of a textarea that grows in a useless way.

Reply:
I think for now we should ignore this. I don't even think we should create an issue for it. The message form is a placeholder implementation and we should just cover this detail as part of #82 rather than creating individual issues for every detail we want the pull request for that issue to cover.